### DISCUSSION TITLE
arendst/Tasmota - Gource visualisation

### DISCUSSION THREAD

**Discussion by TrDA-hab** [arendst/Tasmota#21475] [Show and tell]
I think it's interesting. This is a visualization of the great work of the Tasmota team during the period 2017-2024. Watch it on YouTube.
And no, this was not made by my hands.

[Visualization of the work of the Tasmota team](https://youtu.be/kkQYk75uuIY)

<img width="1026" alt="123" src="https://github.com/arendst/Tasmota/assets/56403720/a47f2fe6-e4cb-4280-af8f-7ad50121abdb">

Best regards,
TrDA.

**Comment by arendst** [comment]
Awesome

**Reply by s-hadinger** [reply]
Impressive!
